src/benchmark_runner.cc

```diff
@@ -179,11 +180,11 @@ class BenchmarkRunner {
 
     // Maybe calculate complexity report
     if ((b.complexity() != oNone) && b.last_benchmark_instance) {
```

Comment:
Yes, `b.last_benchmark_instance` will need fixing too, not here.

Reply:
you can also, i think, replace the check against `b.complexity()` with a check against `complexity_reports != nullptr` which might be a bit simpler to reason about.